auth_admin_passkey/tests/test_auth_admin_passkey.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (C) 2013-Today GRAP (http://www.grap.coop)
+# @author Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import exceptions
+from odoo.tests import common, tagged
+from odoo.tools import config
+
+
+@tagged("post_install", "-at_install")
+class TestAuthAdminPasskey(common.TransactionCase):
+    """Tests for 'Auth Admin Passkey' Module"""
+
+    def setUp(self):
```

Comment:
Could you convert this to class method ?

Reply:
Could you also add this : 

`cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))`